cli/chaosli/README.md

```diff
@@ -0,0 +1,59 @@
+# ChaosCLI
```

Comment:
This should be `Chaosli` if that's the name we're going with. Or title this `Chaos Controller CLI` and then the first paragaph would be, `The Chaos Controller CLI, chaosli, ....`